src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -170,6 +172,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
     [OuterLoop]
     public static void SendTimeout_Zero_Throws_TimeoutException_Immediately()
     {
+        TimeSpan serviceOperationTimeout = TimeSpan.FromMilliseconds(10000);
         BasicHttpBinding binding = new BasicHttpBinding();
```

Comment:
If we're worried about occupying a thread on the server side, then we should make the service code be task based and do an await Task.Delay(timeout). That way we don't occupy a thread while waiting.